docs/software/serial-protocols.md

```diff
@@ -57,3 +57,31 @@ When connected to the receiver with the transmitter, enter the ExpressLRS Lua sc
 <figure markdown>
 <img class="center-img" src="../../assets/screenshots/receiver-serial.png" width="auto">
 </figure>
+
+## SUMD Notes
+
+SUMD is the Graupner HoTT bus protocol to transmit multiple channel data over a single serial line. It transports up to 16 full res channels at a rate of 100Hz and is mainly used to provide digital channel data to Flybarless Control Systems for helicopters and older Flight Controllers.
+
+As already stated many times in this manual ExpressLRS expects AUX1 (channel 5) to control the arm state. This may lead to a conflict using serial protocols like SUMD in conjunction with device having fixed channel assignments. To circumvent the problem SUMD implements a channel remapping by swapping channels 5 and 8 in its serial output:
```

Comment:
"times in this manual, ExpressLRS"
comma

"conjunction with a device" or "conjunction with devices"

"To circumvent the problem, SUMD"
comma

Reply:
done